Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -62,30 +68,51 @@
         'Custom' : 'SLHCUpgradeSimulations/Configuration/combinedCustoms.cust_2023tilted',
         'Era' : 'Phase2C1',
         'ScenToRun' : ['GenSimFull','DigiFull','RecoFullGlobal','HARVESTFullGlobal'],
-    },
+    },    
     '2023D2' : {
         'Geom' : 'Extended2023D2',
         'GT' : 'auto:run2_mc',
         'Custom' : 'SLHCUpgradeSimulations/Configuration/combinedCustoms.cust_2023flat',
         'Era' : 'Phase2C1',
         'ScenToRun' : ['GenSimFull','DigiFull','RecoFullGlobal','HARVESTFullGlobal'],
-    },
+    },    
     '2023D3' : {
         'Geom' : 'Extended2023D3',
         'GT' : 'auto:run2_mc',
         'Custom' : 'SLHCUpgradeSimulations/Configuration/combinedCustoms.cust_2023tilted',
         'Era' : 'Phase2C2',
         'ScenToRun' : ['GenSimFull','DigiFull','RecoFullGlobal', 'HARVESTFullGlobal'],
-    },
+    }    
 }
 
+
+#Timing (later we can alter geometry, etc, if need be)
+upgradeProperties[2023]['2023D1Timing'] = deepcopy(upgradeProperties[2023]['2023D1'])
+upgradeProperties[2023]['2023D1Timing']['Era'] = 'Phase2C1_timing'
+upgradeProperties[2023]['2023D2Timing'] = deepcopy(upgradeProperties[2023]['2023D2'])
+upgradeProperties[2023]['2023D2Timing']['Era'] = 'Phase2C1_timing'
+upgradeProperties[2023]['2023D3Timing'] = deepcopy(upgradeProperties[2023]['2023D3'])
+upgradeProperties[2023]['2023D3Timing']['Era'] = 'Phase2C2_timing'
+
+#standard PU sequences
+upgradeProperties[2023]['2017PU'] = deepcopy(upgradeProperties[2017]['2017'])
+upgradeProperties[2023]['2017PU']['ScenToRun'] = ['GenSimFull','DigiFullPU','RecoFullPU','HARVESTFullPU']
```

Comment:
Yeah, this looks like a rebase mistake - the real 2017PU workflows are defined further up. @lgray, can you fix this in your followup PR? (or @boudoul, if you're rebasing your PR onto this anyway, feel free to fix it there...)